Makefile

```diff
@@ -21,6 +21,12 @@
 #              will be harmless on other machines.
 ######################################################################
 
+LUA_LINK=
```

Comment:
Yes, you are right. I need a few more lines on my local copy but not on Kiwi. We do need them loads, yes. I'll add them in.